src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -509,6 +517,52 @@ def publish_node_launch_events(self, failed_nodes: Dict[str, List[str]]):
             event_supplier=self._flatten_failed_launch_nodes(failed_nodes),
         )
 
+    @log_exception(logger, "publish_job_info_events", catch_exception=Exception, raise_on_error=False)
```

Comment:
Should the name in log_exception match the method name?

Reply:
Good catch, fixed